[2023-03-21 01:29:35,604] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_oetker_app.run_my_script_task manual__2023-03-21T01:29:34.992044+00:00 [queued]>
[2023-03-21 01:29:35,614] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_oetker_app.run_my_script_task manual__2023-03-21T01:29:34.992044+00:00 [queued]>
[2023-03-21 01:29:35,615] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 01:29:35,615] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-21 01:29:35,616] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 01:29:35,633] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_my_script_task> on 2023-03-21 01:29:34.992044+00:00
[2023-03-21 01:29:35,641] {standard_task_runner.py:52} INFO - Started process 1142 to run task
[2023-03-21 01:29:35,645] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_oetker_app', 'run_my_script_task', 'manual__2023-03-21T01:29:34.992044+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/dag_oetker_app.py', '--cfg-path', '/tmp/tmpx5yrhf_5', '--error-file', '/tmp/tmpel0rheji']
[2023-03-21 01:29:35,648] {standard_task_runner.py:80} INFO - Job 11: Subtask run_my_script_task
[2023-03-21 01:29:35,745] {task_command.py:369} INFO - Running <TaskInstance: dag_oetker_app.run_my_script_task manual__2023-03-21T01:29:34.992044+00:00 [running]> on host ec81750b2a36
[2023-03-21 01:29:35,952] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=julia
AIRFLOW_CTX_DAG_ID=dag_oetker_app
AIRFLOW_CTX_TASK_ID=run_my_script_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T01:29:34.992044+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T01:29:34.992044+00:00
[2023-03-21 01:29:36,127] {extract_raw_data.py:20} INFO - success....raw data file saved into /opt/***/output/ folder
[2023-03-21 01:29:36,129] {python.py:173} INFO - Done. Returned value was: None
[2023-03-21 01:29:36,147] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_oetker_app, task_id=run_my_script_task, execution_date=20230321T012934, start_date=20230321T012935, end_date=20230321T012936
[2023-03-21 01:29:36,188] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-21 01:29:36,233] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
